.gitlab/build_quartz.yml

```diff
@@ -39,8 +39,9 @@ quartz_release:
 .build_on_quartz:
   stage: build
   variables:
-    ALLOC_COMMAND: "srun -p pdebug -t 30 -N 1 ${ASSIGN_ID} --interactive"
+    ALLOC_COMMAND: "srun -p pdebug -t 10 -N 1 ${ASSIGN_ID} --interactive"
   extends: [.build_script, .on_quartz]
+  needs: [quartz_allocate]
```

Comment:
This is the triggering fix.  Without this, both allocate jobs were blocking all build jobs (minus lassen because it doesnt have any connection to allocate/release).

Reply:
Thanks for tracking this down @white238 !

Reply:
Yea chalk this up to the category "How did this work at all before"...